webpages/settings/index.js

```diff
@@ -325,7 +325,7 @@ const vue = (window.vue = new Vue({
         vue.sidebarToggle();
       }
       if (this.isOpen) {
-        this.modalToggle;
+        void this.modalToggle;
```

Comment:
This is weird.

Reply:
@GrahamSH-LLK This doesn't appear to do anything, remember what this was meant to do?

Reply:
This shouldn't exist, I don't even remember coding this. Probably some kind of debugging I added.

Reply:
@apple502j Please remove this if statement, it never did anything anyway. `this.modalToggle` is not even a getter (it may be technically, not sure, but Vue won't do anything about it), obtaining its value does not have any direct or side effects.